metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
It's only 4 characters, so I guess sticking to `*_cape_cin` isn't so bad. I see your point with (what should be) `surface_based_cape_cin`, but I think `sb` is enough. There's always the first line docstring to clarify it. Otherwise we'd have:
- `surface_based_cape_cin`
- `mixed_layer_cape_cin`
- `most_unstable_cape_cin`

Hmmm....now I've turned myself around. I almost wonder if those would actually help code clarity. `MLCAPE`, `SBCAPE`, and `MUCAPE` almost seem like ye olde FORTRAN names rather than something we should propagate into 2017.